### ISSUE TITLE
License

### ISSUE THREAD

**Issue by rkdarst** [coderefinery/handsonscicomp#10]
We need a license.  Please comment here on preferences.

* hpc-carpentry uses CC-BY
* Aalto tutorials use CC-BY
* CodeRefinery uses CC-BY-SA
* sw carperntry and others seem to use CC-BY

We can leave this issue open until more contributors join and then decide.  My thought is CC-BY, this is more of basic infra so can be more open.

The sphinx extensions are a separate thing - sphinx itself uses BSD2 so that's what I go with by default.

**Comment by bast** [comment]
> We can leave this issue open until more contributors join and then decide. My thought is CC-BY, this is more of basic infra so can be more open.

I agree. The "SA" part can limit reuse and I think we should maximize reuse.
